buildpackage/config_reader_test.go

```diff
@@ -140,7 +140,7 @@ func testBuildpackageConfigReader(t *testing.T, when spec.G, it spec.S) {
 
 			_, err = packageConfigReader.Read(configFile)
 			h.AssertNotNil(t, err)
-			h.AssertError(t, err, "unknown configuration element")
+			h.AssertError(t, err, "unknown configuration element ")
```

Comment:
Made this test better. If the space had been there before this test would have failed when we started to pluralize everything.